.github/workflows/slm-build.yml

```diff
@@ -0,0 +1,100 @@
+name: SLM / Nusim Build
+
+on:
+  workflow_dispatch:
+#  schedule:
+#    - cron: "0 0 * * *"
+
+jobs:
+  toolchain-check:
+    runs-on: ubuntu-latest
+    outputs:
+      toolchain_id: ${{ steps.docker_image.outputs.TOOLCHAIN_ID }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          path: ncs/nrf
+      - name: Docker image
+        id: docker_image
+        working-directory: ncs/nrf
+        run: |
+          TOOLCHAIN_ID=$(./scripts/print_toolchain_checksum.sh)
+          echo $TOOLCHAIN_ID
+          echo "TOOLCHAIN_ID=$TOOLCHAIN_ID" >> $GITHUB_OUTPUT
+  build:
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/nrfconnect/sdk-nrf-toolchain:${{ needs.toolchain-check.outputs.toolchain_id }}
+      credentials:
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+    needs: toolchain-check
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          path: ncs/nrf
+      - name: West
+        working-directory: ncs/nrf
+        run: |
+          west init -l .
+          west config manifest.group-filter +nusim
+          west update
+      - name: Build SLM nuSIM
+        working-directory: ncs/nrf
+        run: |
+          cat << EOF > app.overlay
```

Comment:
this looks weird.  can we move the nu-sim as an add-on repo that points to sdk-nrf revision with it's own west manifest?